public/app/features/alerting/unified/components/notification-policies/EditNotificationPolicyForm.tsx

```diff
@@ -168,14 +157,25 @@ export const AmRoutesExpandedForm = ({
 
       <Field label="Contact point">
         <Controller
-          render={({ field: { onChange, ref, ...field } }) => (
-            <Select
-              aria-label="Contact point"
-              {...field}
-              className={formStyles.input}
-              onChange={(value) => onChange(mapSelectValueToString(value))}
-              options={receiversWithOnCallOnTop}
-              isClearable
+          render={({ field: { onChange, ref, value, ...field } }) => (
+            <ContactPointSelector
+              selectProps={{
+                ...field,
+                className: formStyles.input,
+                onChange: (value) => {
```

Comment:
It looks like we're no longer using `selectableValuesToStrings`, `selectableValueToString` and `mapSelectValueToString` here so we should be free to remove those.

It does look like the logic in this `onChange` handler is the exact same as in the default policy editor form, perhaps we can abstract this into a separate function?

Reply:
We are using some of them actually :( 
`selectableValueToString` used in `selectableValuesToStrings`, which is used in `mapMultiSelectValueToStrings`, which is used in a few places 😞 

But it seems we can get rid of `mapSelectValueToString`

I'll move that other method into a helper util :+1: